apps/greenhouse/package.json

```diff
@@ -15,10 +15,11 @@
   "private": true,
   "devDependencies": {
     "@cloudoperators/juno-config": "*",
-    "@types/react": "18.3.12",
     "@svgr/plugin-jsx": "^7.0.0",
     "@tailwindui/react": "^0.1.1",
     "@testing-library/react": "^16.0.1",
+    "@types/lodash": "4.17.16",
```

Comment:
(Optional)
So we will use this approach of mapping data?

Reply:
Not sure if I get it completely but this was to fix the failing `typecheck` in `greenhouse` without which I was unable to push my changes. Someone tried to [use](https://github.com/cloudoperators/juno/blob/3cce2def6ff3d48ac92a86ab142e0f5556d8c29b/apps/greenhouse/src/components/core-apps/org-admin/components/teams/hooks/helper/updateTeamMemberships.ts#L6) `lodash` in greenhouse without adding it and it's corresponding types package as dependencies in package.json.